### PR TITLE
/services authentication. r= fabrice

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

**Comment by ferjm** [comment]
@fabricedesre we can now disable authentication at build time by removing the `authentication` feature from the `Cargo.toml` file

File: Cargo.toml [hunk 1]
```diff
@@ -4,12 +4,16 @@ version = "0.1.0"
 authors = ["fabrice <fabrice@desre.org>"]
 build = "build.rs"
 
+[features]
+default = ["authentication"]
+authentication = []
+
```
Comment (fabricedesre) [inline Cargo.toml:10]: Can you add some documentation in the Readme about that?

Comment (ferjm) [inline Cargo.toml:10]: Done

**Comment by fabricedesre** [comment]
r=me. squash & merge!

**Pull request merged**